example/article/models.py

```diff
@@ -1,4 +1,7 @@
-from django.core.urlresolvers import reverse
+try:
+    from django.urls import reverse
+except ImportError:  # Django<2.0
+    from django.core.urlresolvers import reverse
```

Comment:
Can this be moved below the import statements? (since it's code)

Reply:
Done.